<?xml version="1.0"?>
<launch>
  <arg name="output" default="screen"/>
  <arg name="respawn" default="false"/>
  <arg name="start_manager" default="false"/>
  <arg name="start_server" default="false"/>
  <arg name="nodelet_manager" default=""/>
  <arg name="launch-prefix" default=""/>

  <arg name="verbose" default="true"/>
  <arg name="data_dir" value="$(find uwds)/res/data"/>
  <arg name="match_threshold" value="0.85"/>

  <node name="uwds_server"
        pkg="nodelet" type="nodelet"
        args="load uwds/UwdsServerNodelet $(arg nodelet_manager)"
        respawn="$(arg respawn)"
        output="$(arg output)"
        launch-prefix="$(arg launch-prefix)"
        if="$(arg start_server)"/>

  <node name="uwds_kb_lite"
        pkg="uwds" type="uwds_kb_lite.py"
        respawn="$(arg respawn)"
        output="$(arg output)"
        launch-prefix="$(arg launch-prefix)">
    <rosparam subst_value="true">
      data_dir: $(arg data_dir)
      match_threshold: $(arg match_threshold)
      verbose: $(arg verbose)
    </rosparam>
  </node>

</launch>
